hazelcast/src/main/java/com/hazelcast/client/config/impl/ClientDomConfigProcessor.java

```diff
@@ -198,7 +200,7 @@ private void handleConnectionStrategy(Node node) {
         }
```

Comment:
Can you also check `getEvictionConfig`, `handleUserCodeDeployment`, `handleBackupAckToClient`, `getNearCachePreloaderConfig` and `handleConnectionStrategy`?

Also, check out methods in `AbstractDomConfigProcessor`, there's more there to be covered. Feel free to simplify the code a bit if it makes your life easier.

Reply:
Can you still check `getNearCachePreloaderConfig` and `handleConnectionStrategy`?